rollup.config.js

```diff
@@ -1,14 +1,29 @@
-const { defineConfig } = require("rollup");
-const { nodeResolve } = require("@rollup/plugin-node-resolve");
-const typescript = require("@rollup/plugin-typescript");
+import { defineConfig } from "rollup";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import typescript from "@rollup/plugin-typescript";
+import scss from "rollup-plugin-scss";
+import { copy } from "@web/rollup-plugin-copy";
 
-module.exports = () =>
-  defineConfig({
-    input: "src/ose.js",
-    output: {
-      dir: "dist/",
-      format: "es",
-      sourcemap: true,
-    },
-    plugins: [nodeResolve(), typescript()],
-  });
+const staticFiles = ["lang", "packs", "templates"];
+
+export default defineConfig({
+  input: "src/ose.js",
+  output: {
```

Comment:
Can we minify our JS payload?

Reply:
I anticipate a conflict with "double sourcemapping" again if we do

https://stackoverflow.com/a/63235210

Perhaps one of you could confirm?

Reply:
`tsconfig.json` doesn't have `sourcemaps` compiler option defined. Sourcemaps are looking healthy

Reply:
Also, added `rollup-plugin-terser` for minification of JS and `outputStyle: "compressed"` for CSS